pkg/broker/filter/filter_handler.go

```diff
@@ -305,11 +315,16 @@ func (h *Handler) handleDispatchToSubscriberRequest(ctx context.Context, trigger
 	}
 
 	reportArgs := &ReportArgs{
-		ns:          trigger.Namespace,
-		trigger:     trigger.Name,
-		broker:      trigger.Spec.Broker,
-		filterType:  triggerFilterAttribute(trigger.Spec.Filter, "type"),
-		requestType: "filter",
+		ns:            trigger.Namespace,
+		trigger:       trigger.Name,
+		broker:        trigger.Spec.Broker,
+		filterType:    triggerFilterAttribute(trigger.Spec.Filter, "type"),
+		requestType:   "filter",
+		requestScheme: request.URL.Scheme,
```

Comment:
Same as other comments, you'll have to try some other way to get the request scheme